test/Microsoft.AspNetCore.Server.KestrelTests/BadHttpRequestTests.cs

```diff
@@ -52,12 +52,26 @@ public class BadHttpRequestTests
         [InlineData("} / HTTP/1.0\r\n")]
```

Comment:
Is there a test somewhere for `OPTIONS * HTTP/1.1`? Path should be empty and RawTarget should be *.